src/evergreen/build.py

```diff
@@ -81,6 +81,16 @@ def status_counts(self) -> StatusCounts:
         """Get the status counts of the build."""
         return StatusCounts(self.json["status_counts"], self._api)
 
+    def get_human_readable_project_id(self) -> str:
```

Comment:
The problem with this is that every time this function is called, it makes an API call. I could also do the following: 

1. Use an attribute and assign it upon initialization instead
2. Create a function that is called "near" initialization (ie: after Build object is created but before it is returned) to assign the attribute with the result from the Evergreen API call. 

With these implementations, the user can directly use the attribute instead of having to call a function and trigger API calls.

Regarding 1, I know that that we want to avoid setting attribute at initialization, and regarding 2, I don't want to leave gaps where someone may forget to call the function.

Considering that this is a workaround, what is the best thing to do here?